manifests/modelserving/ovms-ootb.yaml

```diff
@@ -16,11 +16,13 @@ objects:
       name: ovms
       annotations:
         openshift.io/display-name: 'OpenVINO Model Server'
-        opendatahub.io/disable-gpu: 'true'
       labels:
         opendatahub.io/dashboard: 'true'
     spec:
       builtInAdapter:
+        env:
+          - name: OVMS_FORCE_TARGET_DEVICE
+            value: NVIDIA
```

Comment:
This should be AUTO per https://docs.openvino.ai/2023.0/notebooks/106-auto-device-with-output.html for supporting both paths